Products/Zuul/facades/devicefacade.py

```diff
@@ -558,30 +558,38 @@ def _moveDevices(self, uids, target):
         devs = (self._getObject(uid) for uid in uids)
         targetname = target.getOrganizerName()
         moved_devices_count = 0
+        success = False
```

Comment:
Is the `success` variable necessary?  I don't see any changes that read the result of `success`.  The Javascript code appears to only care about the `remodel_required` flag.

Reply:
https://github.com/zenoss/zenoss-prodbin/pull/3536/files#diff-b492fee73489e9315626c38afd358b04R994

Reply:
I see now.  I wasn't looking at existing code before.  Only the changed lines.